docs/en/operations/table_engines/jdbc.md

```diff
@@ -0,0 +1,76 @@
+# JDBC
+
+Allows ClickHouse to connect to external databases via JDBC.
+
+To implement JDBC connection, ClickHouse uses the separate program [clickhouse-jdbc-bridge](https://github.com/alex-krash/clickhouse-jdbc-bridge). You should run it as a demon.
+
+This engine supports the [Nullable](../../data_types/nullable.md) data type.
+
+## Creating a Table
+
+```
+CREATE TABLE [IF NOT EXISTS] [db.]table_name  ENGINE = JDBC(dbms_uri, external_database, external_table)
+```
+
+**Engine Parameters**
+
+- `dbms_uri` — URI of an external DBMS.
+
+    Format: `jdbc:<driver_name>://<host_name>:<port>/?user=<username>&password=<password>`.
+    Example for MySQL: `jdbc:mysql://localhost:3306/?user=root&password=root`.
+
+- `external_database` — Database in an external DBMS.
+- `external_table` — A name of the table in `external_database`.
+
+## Usage Example
+
+Creating a table in MySQL:
```

Comment:
There should be a note saying that there's native MySQL engine (with link) and it's preferred, or maybe just change example to some more exotic database.